source/Outfit.cpp

```diff
@@ -232,8 +246,36 @@ void Outfit::Add(const Outfit &other, int count)
 		else
 			oit->second += count * it.second;
 	}
+	for(const auto &it : other.reverseFlareSprites)
+	{
+		auto oit = reverseFlareSprites.begin();
+		for( ; oit != reverseFlareSprites.end(); ++oit)
+			if(oit->first.GetSprite() == it.first.GetSprite())
+				break;
+		
+		if(oit == reverseFlareSprites.end())
+			reverseFlareSprites.emplace_back(it.first, count * it.second);
+		else
+			oit->second += count * it.second;
+	}
+	for(const auto &it : other.steeringFlareSprites)
+	{
+		auto oit = steeringFlareSprites.begin();
```

Comment:
Now you *must* refactor this; it's being copy-pasted multiple times. I posted an example in discord.

Reply:
Created a helper method to reduce the duplicate code.